ci-operator/step-registry/baremetal/lab/ipi/external-lb/baremetal-lab-ipi-external-lb-workflow.yaml

```diff
@@ -3,6 +3,7 @@ workflow:
   steps:
     pre:
       - chain: baremetal-lab-ipi-conf-external-lb
+      - ref: baremetal-lab-ipi-conf-network-vips
```

Comment:
This vips step can be moved to the `baremetal-lab-ipi-conf-external-lb` chain. Also, check if the network information can be deleted from the `baremetal-lab-ipi-conf-external-lb` step and applied from the `baremetal-lab-upi-conf-network` step instead.

Reply:
I moved. baremetal-lab-ipi-conf-network-vips fro WF to chain.

> This vips step can be moved to the baremetal-lab-ipi-conf-external-lb chain. Also, check if the network information can be deleted from the baremetal-lab-ipi-conf-external-lb step and applied from the baremetal-lab-upi-conf-network step instead.

I am not sure it's possible without modifications, if at all.

Reply:
> Also, check if the network information can be deleted from the baremetal-lab-ipi-conf-external-lb step and applied from the baremetal-lab-upi-conf-network step instead.

in baremetal-lab-ipi-conf-network we decide based on pv4_enabled, ipv6_enabled and PRIMARY_NET, which come from test config 

we don't have  such vars or other ones in baremetal-lab-ipi-conf-external-lb.

Reply:
I don’t think baremetal-lab-upi-conf-network is sufficient, since machineNetwork for external-lb looks different:
machineNetwork:
  - cidr: ${INTERNAL_NET_CIDR}
  - cidr: $(yq “.api_vip” “${SHARED_DIR}/external_vips.yaml”)/32
For now we don’t have external-lb dual stack job, so I suggest adding  IPv6 machine CIDR to the external-lb config during the work on that dual stack job in order test these changes.